Doc/library/test.rst

```diff
@@ -691,7 +687,10 @@ The :mod:`test.support` module defines the following functions:
 
 .. function:: disable_faulthandler()
 
-   A context manager that replaces ``sys.stderr`` with ``sys.__stderr__``.
+   A context manager that uses ``sys.__stderr__`` instead of
+   :data:`sys.stderr`, since :mod:`test.regrtest` replaces :data:`sys.stderr`
+   with a :class:`io.StringIO` which has no file descriptor when a test is run
+   with ``-W/--verbose3``.
```

Comment:
Well, no. The purpose of this context manager is to disable temporarily faulthandler.